instrumentation/aws-java-sdk-sqs-2.1.0/src/test/java/com/newrelic/utils/SqsV2UtilTest.java

```diff
@@ -122,7 +122,7 @@ public void testCanAddDtHeaders_SendMessageRequest_EmptyRequest_ReturnTrue() {
     @Test
     public void testCanAddDtHeaders_SendMessageRequest_8AttributesWithBody_ReturnTrue() {
```

Comment:
Nit, rename the methods in this test file to match the updated attribute limit (ie should be `testCanAddDtHeaders_SSendMessageRequest_7AttributesWithBody_ReturnTrue` or `...maxAttributesWithBody...` or similar)

Reply:
I renamed the methods